payment/src/test/java/com/kyriba/conference/payment/PaymentControllerTest.java

```diff
@@ -26,8 +40,13 @@
  * @author Igor Lizura
  */
 @RunWith(SpringRunner.class)
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class PaymentControllerTest {
+    @LocalServerPort
+    private int port;
+
+    @MockBean
+    private PaymentService paymentService;
```

Comment:
For API level tests we don't mock anything. If you need DB, then run it with testcontainers.